.github/workflows/submit-contract-sizes.yml

```diff
@@ -0,0 +1,46 @@
+# The workflow has write access, so it needs to be isolated for security reasons from pull request-based workflows,
+# which may be triggered from forked repositories.
+on:
+  workflow_run:
+    workflows:
+      - continuous-integration
+    types:
+      - completed
+
+jobs:
+  submit-contract-sizes:
+    permissions:
+      pull-requests: write
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    if: >
+      github.event.workflow_run.event == 'pull_request' &&
+        github.event.workflow_run.conclusion == 'success'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 100
+
+      - name: Download Contract Sizes
+        uses: ./.github/download-artifact
```

Comment:
So this action is part of https://github.com/paritytech/ink/pull/1971, why is this a standalone PR if it is depending on stuff in the other PR?

Reply:
I missed it, thanks. I will update this PR